src/js/cards.js

```diff
@@ -42,12 +43,24 @@ function openZoneDeck(zone, activeQuest) {
         };
 
     context.audioPlayer = audioPlayerTemplate(context);
+
     var html = zoneTemplate(context);
     addDeckToPage(html);
 
-    adjustHistoricContentFontSize();
+    var $contextEl = $('#card-holder').find('.historic-context'),
+        $container = $contextEl.parent(),
+        options = {
+            allowUpscale: true,
+            subElement: 'p',
+            offset: 50
+        };
+    // Make the font size of the historic context screen large enough to fill the
+    // appropriate space. This helps make the text look good on various devices.
+    utils.adjustFontSizeToEl($contextEl, $container, options);
     $(window).on('orientationchange', function() {
-        setTimeout(adjustHistoricContentFontSize, 1000);
+        setTimeout(function() {
+            utils.adjustFontSizeToEl($contextEl, $container, options);
+        }, 1000);
```

Comment:
Real devices animate the orientation change and take a bit to redraw afterwards. When I initially tried it, if I grabbed the height right after the change, I was getting the previous orientation height. It took more than a half second but less than a full second to get the new height.